test-app/app/models/blog.rb

```diff
@@ -6,7 +6,13 @@ class Blog < ActiveRecord::Base
   alias_attribute :headline, :title
   alias_attribute :text, :content
 
-  enum status: [:draft, :published, :archived]
+  # Rails 7.0 deprecated defining enums with keyword arguments.
```

Comment:
this comment is still wrong, the old syntax is allowed on 7.0 and 7.1 and has been deprecated in 7.2